nipype/interfaces/ants/visualization.py

```diff
@@ -23,7 +23,7 @@ class ConvertScalarImageToRGBInputSpec(ANTSCommandInputSpec):
                        position=1)
     output_image = traits.Str('rgb.nii.gz', argstr='%s', usedefault=True,
                               desc='rgb output image', position=2)
-    mask_image = File('none', argstr='%s', exists=True,
+    mask_image = File('none', argstr='%s',
```

Comment:
What about:

```Python
mask_image = traits.Either('none', File(exists=True), argstr='%s', ...)
```

That way we get the existence check while allowing 'none'.

Reply:
@sulantha2006, do you think you can modify this in order for us to get it merged? Thanks a lot for the contribution